salesagility/SuiteCRM
# Add Elastic Search Repair option to Admin->Repair Menu

Add Elastic Search Repair Index Option to the Admin->Repair Page

## Description





## Motivation and Context

This solves any possible issues for Elastic Search after a Quick R + R

## How To Test This

1. Go to Admin ->  Repar
2. See new option is there and that it works as expected

## Types of changes


### Final checklist
